test/any/HTMLFileContents-test.js

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
This test doesn't seem worthwhile. It amounts to checking that FOAM's `create()` implementation is correct.

If you intend to guard against mistakenly `final`ing something that may be set, we could switch the test to do `foo.create(); foo.bar = 'bar'; expect(foo.bar).toBe('bar')`, but if that's not what you meant to test, then I'd say we can drop this test entirely.